apps/jubmoji-quest/src/pages/api/cards.ts

```diff
@@ -0,0 +1,40 @@
+import type { NextApiRequest, NextApiResponse } from "next";
+import prisma from "@/lib/prisma";
+
+export default async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse
+) {
+  if (req.method === "POST") {
+    const { index, name, description, owner } = req.body;
+
+    try {
+      // Todo: Need validations for data
```

Comment:
What kind of validation are you thinking about? Won't we just set up the 200 cards once and then never "create" any more cards on the DB? Or is this just exposing an endpoint to do that adding once I get the cards?

Reply:
This validation is just a sanity check for when you add the cards